test/html/template-element.js

```diff
@@ -47,3 +47,11 @@ const docWithTemplateAttribute = parseHTML(`<div template="anything"><p>not insi
 
 assert(docWithTemplateAttribute.querySelector('*').tagName, 'P');
 assert(docWithTemplateAttribute.querySelectorAll('*').length, 1);
+
+
+let {document: document2} = parseHTML('<template><img id="el-14" class="has-svg" src="./favicon.svg" inline=""></template>');
+let template2 = document2.documentElement;
+assert(template2.innerHTML, '<img id="el-14" class="has-svg" src="./favicon.svg" inline="">');
+
+template2.content.childNodes[0].replaceWith(document.createElement('br'))
```

Comment:
if you save

```
const content = template2.content
```

`content.toString()` will give the correct content.

`template2.content.innerHTML` still won't.

Reply:
yeah, live nodes are not a thing in here ... and won't be because that's not a goal of this project.

Reply:
To clarify: I don't know if `content` returns an array of its nodes but if that's the case, that's gonna be the case in here. For everything else, JSDOM got you covered. This project is not a replacement for it, it just enables the read-only things a page would land, everything else really need a good use case for it or it won't happen. Do you want to dig into the `DocumentFragment` hole without affecting performance? Please do, if that's the case, I'll be watching 👍

Reply:
what should I do instead?

the idea is to swap out `<a/>` with `<c/>` without affecting anything else.

```
<div>
<a/>
<b/>
</div>
```

Reply:
I found the solution. The problem is not about live nodes. 

`template.querySelectorAll` works.

`template.content.querySelectorAll` doesn't. `template.content` **clones each node**.

Reply:
yeah, `templte.content` does that and it's **not a LIVE NodeList** so you got your answer ... that won't land in here, it's no purpose whatsoever, so if you solved, remove the failing-but-not-really test and let's merge your changes?